src_modules/player/index.ts

```diff
@@ -10,7 +7,7 @@ import { Guild, GatewayIntentBits, VoiceState } from "discord.js";
 import { Bot, CommandGroup, ICommandContext, IExtendedGuild, Logger, Module } from "../..";
 
 import { ClearCommand } from "./commands/ClearCommand";
-import { LyricsCommand } from "./commands/LyricsCommand";
```

Comment:
Revert the removal of the lyrics command import. Both the lyrics command and the pop command should be imported.